### ISSUE TITLE
Implement compatibility with hidden information games

### ISSUE THREAD

**Issue by Neoflash1979** [Neoflash1979/macao#9]

**Comment by Neoflash1979** [comment]
@flesler Yes, I think poker would be a good example of hidden information. At any given time, a player is unaware of the cards in other players' hands and of what cards are left in the deck and in what order. A player is only aware of his own cards and the cards that have been flipped on the table (in the case of Texas Hold'em).

**Comment by pickhardt** [comment]
Hey, just wondering if you made anything related to this that wasn't merged yet?

**Comment by snowfrogdev** [comment]
> Hey, just wondering if you made anything related to this that wasn't merged yet?

No, I haven't done anything with this repo in 5 years.
